js/clue.js

```diff
@@ -1,3 +1,10 @@
 export class Clue {
-
+  constructor(question, answer, pointValue, category) {
+    this.question = question;
+    this.answer = answer;
+    this.available = true;
+    this.value = pointValue;
+    this.category = category;
```

Comment:
This looks like a good use-case for destructuring...you could pass in an entire clue object and destructure each of these properties out of there, instead of passing them all in individually. e.g. 

```
constructor(clue) {
    let { question, answer, pointValue, category } = clue;
    this.question = question;
    this.answer = answer;
    etc.
```


Or you could take it one step further and use `Object.assign()` to add all of those properties in one fell swoop:

```Object.assign(this, clue);```